addon/components/bs-navbar.js

```diff
@@ -211,15 +208,19 @@ export default class Navbar extends Component {
    * @event onCollapse
    * @public
    */
-  onCollapse() {}
+  get onCollapse() {
+    return this.args.onCollapse ?? (() => {});
+  }
```

Comment:
Why do we need a noop function as a default here? Can't we change the invoking side to only invoke it if it is provided?

Same question applies to `onCollapsed`, `onExpand`, and `onExpanded`.

Reply:
Good point! I hadn't even considered that as I was too focused on matching exactly what the old `@defaultValue` decorator did.
Removing these fields and invoking them conditionally also will get rid of some documentation however. If we were to add Typescript to this project, we would be able to place that in the Arguments/Signature interface, where it would make more sense, I think.